README.md

```diff
@@ -2,9 +2,45 @@
 ### Brought to you by The Codebenders
 ***
 
+
 ## Code Locations
-* **Server Code**, built in Java Servelets, lives in /server
+* **Server Code**, built in Java Servlets, lives in /server
 * **Web Client Code**, built in React, lives in /frontend
 
+
+## Running Locally
+Both the frontend and backend development servers can be run together
+through the command `bash run-dev.sh` (assuming that one has installed
+all frontend packages with `npm install` and has set their gcloud project
```

Comment:
If you want an extension project, you could add a shell script that does this part as well. That'll speed up the process of switching branches.

Reply:
That's a great idea! I've modified run-dev.sh to take in an optional `-setup` (optionally, just `-s`) flag which runs these one-time commands.